packages/massa-proto-cli/src/protobuf.ts

```diff
@@ -88,5 +88,6 @@ export async function getProtoFunction(
     funcName: funcName,
     resType: returnType,
     protoData: protoFileContent,
+    protoPath: protoFilePath,
```

Comment:
Wait for my PR to be merged, #222, already added tihs field with another name, once my branch is merge rebase on main (also true for other PR on massa-proto-cli)